src/components/navigation/test/Link-test.js

```diff
@@ -0,0 +1,77 @@
+import { mount } from 'enzyme';
+import { createRef } from 'preact';
+
+import Link from '../Link';
+
+describe('Link', () => {
+  const createComponent = (props = {}) => {
+    return mount(
+      <Link href="http://www.example.com" {...props}>
+        This is content inside of a Link
+      </Link>
+    );
+  };
+
+  // Tests common to presentational components
```

Comment:
All tests in this body of work are unrolled and wet, but I have noted groups of tests for presumable extraction.